tests/generative/variable_strategies.py

```diff
@@ -70,6 +70,7 @@ class DomainConstraint:
             value: ({int, float, bool, datetime.date}, DomainConstraint.PATIENT),
             select_column: ({int, float, bool, datetime.date}, DomainConstraint.ANY),
             eq: ({bool}, DomainConstraint.ANY),
+            is_null: ({bool}, DomainConstraint.ANY),
```

Comment:
I think maybe I'd argue that `IsNull` is "simpler" than `Eq`, but I don't feel strongly.